### DISCUSSION TITLE
Can kiali  support skywalking for tracing?

### DISCUSSION THREAD

**Discussion by kubehan** [kiali/kiali#7124] [Q&A]
### What do you want to discuss?
Can kiali  support skywalking for tracing?
### What are the alternate solutions?

**Comment by jmazzitelli** [comment]
_moving to Q&A discussion_

> Can kiali support skywalking for tracing?

If Skywalking requires its own client and has its own client API, then no. If it conforms to the Jaeger/Tempo tracing format, then you can just configure the Kiali tracing settings to point to your Skywalking tracing endpoint as if it was a Jaeger or Tempo backend.
